### PR TITLE
Update on DG and UG

### PR DESCRIPTION
**Pull request by huanghao1998**
What's change
- Add remove feature under DG 's implementation
- Add clearpj command in UG
- Add UI annotation picture
- Update UiClassDiagram
- Update main UI picture
- Update HelpWindow picture
- Update sizing for the GUI
- Update personal repoSense link
- Update all the screenshot

**DONE**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -306,6 +306,28 @@ If no argument is provided, an empty list will be shown.
...
+The feature is facilitated by the `RemoveCommand` class and `RemoveProjectCommand` classes.
+They extend `Command` and implements the following operations:
+* `RemoveCommand#execute()`  — Removes the respective Person's field value based on the given input.
+* `RemoveProjectCommand#execute()`  — Removes the respective Project's field value based on the given input.
+
+The `RemoveCommandParser` and `RemoveProjectCommandParser` classes are used to parse the user input and removes the value that matches to the existing value in the respective field using `RemovePersonDescriptor` and `RemoveProjectDescriptor` respectively.
+`RemovePersonDescriptor` and `RemoveProjectDescriptor` objects are created and pass together with the `Index` (parses from the user input) into respective commands.
+The respective classes verify that the user input is valid and create the commands accordingly.
+
+If no argument is provided, exception will be thrown. Else if only predicate is present, the corresponding field of the Person/Project will be emptied.
```
Comment (dillongoh) [inline docs/DeveloperGuide.md:322]: Can reconsider some of the phrasing. Eg. "an exception will be thrown", "Otherwise, if only predicate...."

**Review by dillongoh — COMMENTED** [review]
Some grammatical and phrasing issues to consider.

**Review by nus-jeff — APPROVED** [review]
LGTM!

**Review by nus-jeff — APPROVED** [review]
LGTM 👍 Thanks for standardizing the image size.

**Pull request merged**